corehq/apps/domain/views/settings.py

```diff
@@ -491,6 +492,13 @@ def get_success_url(self):
                     reverse('domain_login', args=[couch_user.domain]),
                     couch_user.raw_username,
                 )
+            elif couch_user:
+                return super().get_success_url()
+            try:
+                ConsumerUser.objects.get(user=self.user)
+                return reverse('consumer_user:patient_login')
+            except ConsumerUser.DoesNotExist:
```

Comment:
It looks like there are a lot of these places where you `pass` when one of these doesn't exist. You could use a `get_or_none` type manager to clean up a lot of this code eg: 

https://github.com/dimagi/commcare-hq/blob/5c9a9144182a58b31d2a356105d2d88b96506fc3/corehq/apps/case_search/models.py#L25-L34

Reply:
Added the object manager to one model and made changes accordingly for better readability